packages/core/README.md

```diff
@@ -5,13 +5,50 @@ interfaces in dhis2 applications.
 
 ## Installation
 
-```
-yarn add @dhis2/d2-ui-core
+```sh
+> yarn add @dhis2/d2-ui-core
 ```
 
 ## Usage
 
 Importing components
+
+```js
+import Button from "@dhis2/d2-ui-core/Button";
```

Comment:
I think its actually `import Button from "@dhis2/d2-ui-core/button/Button`;

This is because I didn't get time to rename files to index.js.  I do have a partially implemented branch for that though.

Reply:
Ok nice catch, I actually just guessed on that syntax since I was updating the README anyway.

Reply:
Fixed